srcpkgs/gnome-bluetooth/template

```diff
@@ -1,21 +1,21 @@
 # Template file for 'gnome-bluetooth'
 pkgname=gnome-bluetooth
-version=3.34.5
+version=42.0
 revision=1
 build_helper="gir"
 build_style=meson
 configure_args="-Dgtk_doc=false -Dintrospection=$(vopt_if gir true false)"
 hostmakedepends="pkg-config intltool itstool $(vopt_if gir gobject-introspection) glib-devel"
-makedepends="libXi-devel gtk+3-devel libnotify-devel dconf-devel
- gvfs-devel bluez eudev-libudev-devel libcanberra-devel"
+makedepends="libXi-devel gtk4-devel libadwaita-devel libnotify-devel dconf-devel
```

Comment:
the -devel package dependencies have to be changed too

Reply:
Wdym?

Reply:
Nvm, I see what you mean. Thanks.